omnigibson/object_states/overlaid.py

```diff
@@ -62,15 +62,18 @@ def _get_value(self, other):
 
         # Compute the base aligned bounding box of the rigid object.
         bbox_center, bbox_orn, bbox_extent, _ = other.get_base_aligned_bbox(xy_aligned=True)
-        vertices_local = np.array(list(itertools.product((1, -1), repeat=3))) * (bbox_extent / 2)
-        vertices = trimesh.transformations.transform_points(vertices_local, T.pose2mat((bbox_center, bbox_orn)))
+        vertices_local = th.tensor(list(itertools.product((1, -1), repeat=3))) * (bbox_extent / 2)
+        vertices = th.tensor(
+            trimesh.transformations.transform_points(vertices_local, T.pose2mat((bbox_center, bbox_orn))),
+            dtype=th.float32,
+        )
```

Comment:
Does `trimesh.transformations.transform_points` accept torch tensors as inputs?

In general, maybe it's a good time to get rid of `trimesh.transformations` from our codebase in favor of our own `utils.tranform_utils`?